analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -766,9 +767,13 @@ def signal_handler(signum, frame):
     for skp in skipped_actions:
         LOG.debug_analyzer("%s is skipped", skp.source)
 
-    LOG.info("Total analyzed compilation commands: %d", len(analyzed_actions))
-    if skipped_actions:
-        LOG.info("Skipped compilation commands: %d", len(skipped_actions))
+    LOG.info("Total analyzed compilation commands: %d", cmp_cmd_count.analyze)
+    # Some compule commands are skipped during log processing
+    # if nothing is skipped there for pre-analysis step,
```

Comment:
It would be more clear to put it this way:
Some compile commands are skipped during log processing. If nothing is skipped...